ksql-rest-app/src/main/java/io/confluent/ksql/rest/server/KsqlServerMain.java

```diff
@@ -99,4 +104,28 @@ private static Executable createExecutable(
     builder.put(StreamsConfig.APPLICATION_ID_CONFIG, KSQL_REST_SERVER_DEFAULT_APP_ID);
     return builder.build();
   }
+
+  // package-private for test purpose
```

Comment:
Replace with the `VisibleForTesting` annotation?

Reply:
Using `VisibleForTesting` annotation now.